catalog/rocks/entry_catalog.go

```diff
@@ -67,16 +69,37 @@ type EntryCatalog struct {
 	store graveler.Graveler
 }
 
+const (
+	rangeAllocationPercent     = 0.8
+	metaRangeAllocationPercent = 1 - rangeAllocationPercent
```

Comment:
These should be parameters in `config`.  Otherwise we need to release to change them.  Different values may be useful to different customers.  Also I think 0.8 is *much* too small -- and just having this discussion means it's hardly a constant.

Reply:
Also these are proportions not percents.

Reply:
Done.